modules/backend/behaviors/relationcontroller/partials/_manage_list.htm

```diff
@@ -21,6 +21,9 @@ <h4 class="modal-title"><?= e(trans($relationManageTitle, [
                     class="btn btn-primary"
                     data-request="onRelationManageAdd"
                     data-dismiss="popup"
+                    data-request-success="
+                        $.oc.relationBehavior.changed('<?= e($this->vars['relationField']) ?>', 'added')
```

Comment:
make this inline please

Reply:
Done in 46d1e0f1611ceee10cd5d920e2690fa8b62465b5.